taipy/gui/_renderers/factory.py

```diff
@@ -597,6 +598,20 @@ class _Factory:
             ]
         )
         ._set_propagate(),
+        "Progress": lambda gui, control_type, attrs: _Builder(
+            gui=gui,
+            control_type=control_type,
+            element_name="Progress",
+            attributes=attrs,
+        )
+        .set_value_and_default(var_type=PropertyType.dynamic_number, with_default=False)
```

Comment:
why specify `with_default=False` ?

Reply:
If I'm correct, then, `with_default=False` is used when the user is passing no default param, right? That's why I set it like this, do I've to change the `set_value_and_default()` param?